spring-integration-http/src/test/java/org/springframework/integration/http/config/HttpOutboundChannelAdapterParserTests-context.xml

```diff
@@ -16,8 +16,12 @@
 
 	<outbound-channel-adapter id="restTemplateConfig" url="http://localhost/test1" channel="requests" rest-template="customRestTemplate"/>
 
+	<outbound-async-channel-adapter id="asyncRestTemplateConfig" url="http://localhost/test1" channel="requests" rest-template="asyncRestTemplate"/>
```

Comment:
Yeah... There is no `outbound-async-channel-adapter` definition in the `spring-integration-http-5.0.xsd` yet...